### ISSUE TITLE
Loading the GitExtensions.sln resets the Exception settings

### ISSUE THREAD

**Issue by mstv** [gitextensions/gitextensions#6606]
## Current behaviour

Every time I load the `GitExtensions.sln`, the `Exception Settings` of the Visual Studio Community Edition are reset to defaults (which do not make much sense to me).

## Expected behaviour

Unless this is an intentional limitation by MS, all exceptions but this threading and ConEmu XML stuff should be enabled.

## Steps to reproduce

- open GitExtensions.sln 
- select all exceptions
- close MSVS
- reopen GitExtensions.sln

## Did this work in previous version of GitExtensions

unknown

## Environment

MSVS Community Edition 2017 and 2019

**Comment by RussKie** [comment]
Could you please provide more info, I don't quite understand the nature of the issue.

**Comment by illfated** [comment]
Related to this, maybe? https://dailydotnettips.com/exception-settings-window/

**Comment by mstv** [comment]
Yes, Trond, this is the feature, although I don't even use these conditions. I simply tell the debugger to stop on all exceptions except some uncritical. This selection is not persisted over the restart of MSVS or it is reset to defaults (which are not very helpful).

- switch on every exception before running GE:
![grafik](https://user-images.githubusercontent.com/36601201/58055875-78c81980-7b5f-11e9-80fa-297c7b3ac37a.png)
- debug GE
- switch off usual exceptions when they are thrown, BindingFailure, AsynchronousThreadAbort, ThreadAbortException, DirectoryNotFoundException, BadImageFormatException, OperationCanceledException:
![grafik](https://user-images.githubusercontent.com/36601201/58056209-9f3a8480-7b60-11e9-93fd-9aa7c33535f9.png)
- exit GE:
![grafik](https://user-images.githubusercontent.com/36601201/58056361-27208e80-7b61-11e9-826c-8e5b372aecee.png)
- close and reopen the GitExtensions.sln:
![grafik](https://user-images.githubusercontent.com/36601201/58056543-c6de1c80-7b61-11e9-8b80-dad97d94dd6b.png)

**Comment by illfated** [comment]
Interesting. That is a lot more details than I expected, but I suppose it is useful in this context.
